crates/moongraph/Cargo.toml

```diff
@@ -20,7 +20,7 @@ tutorial = ["dep:aquamarine"]
 
 [dependencies]
 aquamarine = { version = "0.4.0", optional = true }
-broomdog = { version = "0.1.1", path = "../../../broomdog" }
+broomdog = { version = "0.1.1" }
```

Comment:
Did you run into an error with this? Maybe I had erroneously assumed that this would only use the path if it existed, but would otherwise load from crates.io?

Reply:
Ah, you know this is probably just left over from when this crate lived in a workspace with others.

Reply:
When building locally / from git, it didn't work as it didn't find it at the path.